### PR TITLE
ToDoList: JavaScript Best Practcises

### PR DESCRIPTION
**Pull request by apt2tell**
## To Do List App

> Here, I copied data from ToDoList App for JavaScript best practices inspection.

### CODE REVIEW

File: src/modules/addEvent.js [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+         localStorage.setItem('data', JSON.stringify(data));
+       });
+     });
+
+     for (let i = 0; i < ellipse.length; i += 1) {
+       ellipse[i].addEventListener('click', () => {
+         del[i].style.display = 'inline';
+         ellipse[i].style.display = 'none';
+       });
+     }
```
Comment (Fayob) [inline src/modules/addEvent.js]: I would suggest you use array method like 'forEach' to loop through the ellipse instead of traditional for loop

**Review by Fayob — CHANGES_REQUESTED** [review]
@Okechukwu-muokwugwo
 
- Great effort towards the project

## Changes Required ♻️ 
- Clean Code

There is a change required, kindly check the comment under your code to see the changes required

File: src/modules/addEvent.js (hunk 1 above)
Comment (apt2tell) [reply src/modules/addEvent.js]: @Fayob Thanks 👍

**Review by Fayob — APPROVED** [review]
Your project is complete! There is nothing else to say other than... it's time to merge it :shipit:
Congratulations! 🎉

## Highlights
No linter errors. ✔️
GitHub flow is followed. ✔️
Good Javascript practice. ✔️

**Pull request merged**